src/include/uapi/drm_local/amdxdna_accel.h

```diff
@@ -163,6 +163,41 @@ struct amdxdna_ctx_param_config_cu {
 	struct amdxdna_cu_config cu_configs[];
 };
 
+enum fw_buf_type {
+	debug_buffer = 0,
+	trace_buffer,
+	dbg_queue,
+	log_buffer
+};
+
+/**
+ * struct uc_info_entry: Holds uc index & buffer size allotment info
+ * @index: uc index
+ *     On aie2ps, uc index is same to column index
+ *     On aie4, uc index is mapped as 0->0_A, 1->0_B, 2->1_A, 3->1_B, 4->2_A, 5->2_B
+ * @size: buffer size in words for this uc
```

Comment:
Buffer size in bytes seems more generic and less confusion. Can we change it to bytes?

Reply:
Hi @maxzhen yes, I updated buffer size as bytes in comments. Please check, thanks.